test/e2e/go.mod

```diff
@@ -34,15 +34,17 @@ replace (
 
 require (
 	github.com/google/go-cmp v0.5.5
-	github.com/onsi/ginkgo v1.14.0
-	github.com/onsi/gomega v1.10.1
+	github.com/onsi/ginkgo v1.16.4
+	github.com/onsi/gomega v1.14.0
+	github.com/ovn-org/ovn-kubernetes/go-controller v0.0.0-20220427175813-acd09730af95
 	github.com/pkg/errors v0.9.1
-	k8s.io/api v0.22.2
-	k8s.io/apimachinery v0.22.2
-	k8s.io/client-go v0.22.2
+	golang.org/x/sync v0.0.0-20210220032951-036812b2e83c
+	k8s.io/api v0.23.3
+	k8s.io/apimachinery v0.23.3
+	k8s.io/client-go v0.23.3
 	k8s.io/klog v1.0.0
 	k8s.io/kubectl v0.22.2 // indirect
 	k8s.io/kubelet v0.22.2 // indirect
 	k8s.io/kubernetes v1.22.2
-	k8s.io/utils v0.0.0-20210930125809-cb0fa318a74b
+	k8s.io/utils v0.0.0-20211116205334-6203023598ed
```

Comment:
are you pulling in any changes to this module?

Reply:
gone after the changes